README.md

```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
 * binutils (using `objcopy`)
 * Go >= 1.16
 
+## Available sbctl Packages
+
+List of existing sbctl packages can be found
+[here](https://repology.org/project/sbctl/versions).
```

Comment:
Don't use "here" for hyperlinks:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#use_clear_link_wording

```suggestion
You can find a list of
[sbctl packages on Repology](https://repology.org/project/sbctl/versions).
```